app/components/referee_interface/feedback_hints_component.html.erb

```diff
@@ -4,10 +4,11 @@
   <% if @academic %>
     <li>academic skills</li>
   <% end %>
-  <li>communication skills</li>
-  <li>reliability and professionalism</li>
-  <li>ability to work with children</li>
-  <li>transferable skills</li>
+  <li>Communication skills</li>
+  <li>Reliability and professionalism</li>
+  <li>Transferable skills</li>
+  <li>Ability to work with children</li>
+  <li>Academic skills</li>
```

Comment:
This is already conditionally included in line 5. Should the conditional addition be moved here instead maybe?

Reply:
We decided to not make this conditional, and instead always include it as the last item (as non-academic references may still wish to comment on academic skills).

Also the bullet points should start with lowercase letters, as they follow on from a lead-in line - see [prototype](https://github.com/DFE-Digital/apply-for-teacher-training-prototype/blob/main/app/views/reference/comments.html#L23-L31).